engine/src/main/java/org/terasology/config/AudioConfig.java

```diff
@@ -31,39 +31,63 @@
 
     private transient PropertyChangeSupport propertyChangeSupport = new PropertyChangeSupport(this);
 
+    /**
+     * @return Returns a float of the sound volume
+     */
     public float getSoundVolume() {
         return soundVolume;
     }
 
+    /**
+     * @param soundVolume Sets the sound volume starting from the previous value and going to the soundVolume passed in
+     */
     public void setSoundVolume(float soundVolume) {
         float oldValue = this.soundVolume;
         this.soundVolume = soundVolume;
         propertyChangeSupport.firePropertyChange(SOUND_VOLUME, oldValue, soundVolume);
     }
 
+    /**
+     * @return Returns music volume
+     */
     public float getMusicVolume() {
         return musicVolume;
     }
 
+    /**
+     * @param musicVolume Sets music volume using the same principle as setSoundVolume()
+     */
     public void setMusicVolume(float musicVolume) {
         float oldValue = this.musicVolume;
         this.musicVolume = musicVolume;
         propertyChangeSupport.firePropertyChange(MUSIC_VOLUME, oldValue, musicVolume);
     }
 
+    /**
+     * @return A boolean indicating if sound is disabled
```

Comment:
Boolean getters are an exception - they should be `@return whether sound is disabled`...